trailblazer/store/base.py

```diff
@@ -0,0 +1,17 @@
+from alchy import Query
+from dataclasses import dataclass
+from typing import Type
+
+from trailblazer.store.models import Model
+
+
+@dataclass
+class BaseHandler_2:
```

Comment:
Is this a temporary class that will be refactored soon? I don't think we should have names like this, maybe the existing base handler class can be updated instead?

Reply:
Yes, exactly - it is temporary. Sine the current handler in api.py is called Basehandler, but contains everything. Hoping to move that to crud modules and then rename this.